docs/developer.md

```diff
@@ -92,8 +92,9 @@ is needed. See [Installing Bazel on Windows](https://docs.bazel.build/versions/m
 for more details. Install the following packages:
 
 ```
-pacman -S patch realpath
+pacman -S patch coreutils
 ```
+`realpath` command should be found in shell command line.
```

Comment:
Nit: perhaps:
"Once `coreutils` is installed, the `realpath` command should be present in your shell's path."

Or maybe just omit this? If we are installing the right package above, won't all the right things happen already?

Reply:
I revised it according to your comment.